packages/team-management/src/components/TeamsView/tests/index.test.tsx

```diff
@@ -0,0 +1,113 @@
+import { Provider } from 'react-redux';
+import { store } from '@opensrp/store';
+import { mount, shallow } from 'enzyme';
+import { history } from '@onaio/connected-reducer-registry';
+import { Router } from 'react-router';
+import React from 'react';
+import TeamsView, { defaultProps, loadSingleTeam } from '..';
+import { act } from 'react-dom/test-utils';
+import flushPromises from 'flush-promises';
+import fetch from 'jest-fetch-mock';
+import { org1, teamMember } from '../../../ducks/tests/fixtures';
+import { notification } from 'antd';
+import { ERROR_OCCURRED } from '../../../constants';
+
+describe('components/TeamsView', () => {
+  beforeEach(() => {
+    fetch.resetMocks();
```

Comment:
its preferable to use clearmocks for future tests instead of resetting everything.